MAiD - Case Management App/force-app/main/default/classes/GetCasePHNandVerify.cls

```diff
@@ -57,11 +58,14 @@ public class GetCasePHNandVerify {
                             caseList[0].Warning_Message_If_Any__c = 'There is an existing case in the database with the same PHN';
                         }else if(maidPlanningCaseType.contains(cs.Type)){
```

Comment:
instead of else if , use the above if and add the condition (maidPlanningCaseType.contains(cs.Type in OR ,

and for the nested if , write seperate if statement with (maidPlanningCaseType.contains(cs.Type as and condition,

Reply:
Done as per request